e2e/serial/swap/1_swapFlow1.test.ts

```diff
@@ -364,7 +364,7 @@ it('should be able to open press max on token to sell input', async () => {
     id: `${SWAP_VARIABLES.ETH_MAINNET_ID}-token-to-sell-swap-token-input-swap-input-mask`,
     driver,
   });
-  expect(ethValueBeforeGas).toEqual('10000');
+  expect(ethValueBeforeGas.startsWith('9999.99')).toBe(true);
```

Comment:
Tweaked this a bit since we're subtracting the balance by the gas fee [here](https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/hooks/swap/useSwapInputs.ts#L157-L161)